helm/dns-operator-azure/templates/deployment.yaml

```diff
@@ -24,6 +24,8 @@ spec:
         runAsNonRoot: true
         runAsUser: {{ .Values.pod.user.id }}
         runAsGroup: {{ .Values.pod.group.id }}
+        seccompProfile:
+          type: RuntimeDefault
```

Comment:
We have it for container but it is not enough for PSPs. https://github.com/giantswarm/dns-operator-azure/pull/109/files#diff-959a1b78856f281153d1a550897c1b2fa132cf0fec6000b2f0ecc2bb188d385bR45